run/mosquitto2john.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# This software is Copyright (c) 2021, Blackfell <github at blackfell.net>
+# and it is hereby released to the general public under the following terms:
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted, provided that the above copyright notice
+# appears in all copies and that both that copyright notice and this
+# permission notice appear in supporting documentation.
```

Comment:
This is very similar to 2-clause BSD, but not the same wording. If you insist on stipulating those conditions, then please use 2-clause BSD verbatim - see it e.g. in `src/yescrypt/sha256.h`.

Reply:
Or maybe you did copy this entire license text from somewhere? If so, where? To me, it looks like hybrid of our preferred 0-clause BSD (the usual BSD license cut after the word "permitted") and MIT license.

Reply:
No problem, you're right on that, I think this was incorrectly pulled from 1password2john when I was looking over how the other scripts are put together, I'll use the BSD wording I see in 1password2john if you're content? Reads as follows:

# Permission to use, copy, modify, and/or distribute this software for any
# purpose with or without fee is hereby granted, provided that the above
# copyright notice and this permission notice appear in all copies.

Reply:
That's ISC license, which for that source file we inherited from somewhere else. If you insist, we can accept that indeed (and not being a hybrid it's better than what you had before), but our preferred license wording is found e.g. in `restic2john.py` (lacks the restrictions).

Reply:
Got it, yeah I'm happy with that license and will go with that.